lib/exq/redis/job_stat.ex

```diff
@@ -186,4 +186,31 @@ defmodule Exq.Redis.JobStat do
         val
     end
   end
+
+  def get_redis_commands(namespace, node_id, started_at, master_pid, queues, work_table, poll_timeout) do
+    name = redis_worker_name(namespace, node_id)
+    [
+      ["DEL", "#{name}:workers"],
```

Comment:
@ondrejbartas Hi at the beginning I don't have access o pid of manager server. Can we send this line `["DEL", "#{name}:workers"]` just with other heartbeat messages?